### ISSUE TITLE
Oracle: Unparsable section 'NOMAXVALUE' on CREATE SEQUENCE

### ISSUE THREAD

**Issue by aiprat** [sqlfluff/sqlfluff#6594]
### Search before asking

- [x] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

`NOMAXVALUE` in CREATE SEQUENCE segemnt is unparsable.

### Expected Behaviour

'NOMAXVALUE' in CREATE SEQUENCE can be parsed correctly.

### Observed Behaviour

Unparsable section parsing violation is raised.

==== parsing violations ====
L:   1 | P:  71 |  PRS | Line 1, Position 71: Found unparsable section: 'NOMAXVALUE
                       | NOCYCLE;'
WARNING: Parsing errors found and dialect is set to 'oracle'. Have you configured your dialect correctly?

Syntax tree:
[L:  1, P:  1]      |file:
[L:  1, P:  1]      |    statement:
[L:  1, P:  1]      |        create_sequence_statement:
[L:  1, P:  1]      |            keyword:                                          'CREATE'
[L:  1, P:  7]      |            whitespace:                                       ' '
[L:  1, P:  8]      |            keyword:                                          'SEQUENCE'
[L:  1, P: 16]      |            whitespace:                                       ' '
[L:  1, P: 17]      |            sequence_reference:
[L:  1, P: 17]      |                naked_identifier:                             'TBL_SERVICEALERTMESSAGE_S'
[L:  1, P: 42]      |            whitespace:                                       ' '
[L:  1, P: 43]      |            create_sequence_options_segment:
[L:  1, P: 43]      |                keyword:                                      'INCREMENT'
[L:  1, P: 52]      |                whitespace:                                   ' '
[L:  1, P: 53]      |                keyword:                                      'BY'
[L:  1, P: 55]      |                whitespace:                                   ' '
[L:  1, P: 56]      |                numeric_literal:                              '1'
[L:  1, P: 57]      |            whitespace:                                       ' '
[L:  1, P: 58]      |            create_sequence_options_segment:
[L:  1, P: 58]      |                keyword:                                      'START'
[L:  1, P: 63]      |                whitespace:                                   ' '
[L:  1, P: 64]      |                keyword:                                      'WITH'
[L:  1, P: 68]      |                whitespace:                                   ' '
[L:  1, P: 69]      |                numeric_literal:                              '1'
[L:  1, P: 70]      |    whitespace:                                               ' '
**[L:  1, P: 71]      |    unparsable:                                               !! Expected: 'Nothing else in FileSegment.'**
[L:  1, P: 71]      |        word:                                                 'NOMAXVALUE'
[L:  1, P: 81]      |        whitespace:                                           ' '
[L:  1, P: 82]      |        word:                                                 'NOCYCLE'
[L:  1, P: 89]      |        semicolon:                                            ';'
[L:  1, P: 90]      |    newline:                                                  '\n'
[L:  2, P:  1]      |    [META] end_of_file:

### How to reproduce

Run `sqlfluff parse --dialect oracle <filename>` on file with content

```
CREATE SEQUENCE TBL_SERVICEALERTMESSAGE_S INCREMENT BY 1 START WITH 1 NOMAXVALUE NOCYCLE;
```

### Dialect

oracle

### Version

sqlfluff version
3.3.0

python --version
Python 3.13.1

### Configuration

No configuration

### Are you willing to work on and submit a PR to address the issue?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)

**Comment by aiprat** [comment]
This also applies for `NOMINVALUE`.

I noticed that `NO MAXVALUE` is allowed with CREATE SEQUENCE but not `NOMAXVALUE`.
